Rawl_B502/mínugud_senchusa_laigin_and_so_sís.ttl

```diff
@@ -0,0 +1,1264 @@
+      rdfs:comment "na secht n-epscuip ó Druim Aurchaille"
+    ];
+    rel:ancestorOf [
+      a foaf:Person;
+      foaf:title "Epscop"@sga, "Bishop"@eng;
+      rdfs:comment "na secht n-epscuip ó Druim Aurchaille"
+    ];
+    rel:ancestorOf [
+      a foaf:Person;
+      foaf:title "Epscop"@sga, "Bishop"@eng;
+      rdfs:comment "na secht n-epscuip ó Druim Aurchaille"
+    ].
+
+<#Báethallaig>
+    a foaf:Person;
+    irishRel:genName "Báethallaig";
+    irishRel:nomName "Báethallach";
+    rdfs:comment "Fine Cilli Bicsige";
+    rel:descendantOf <#Conall>.
+
+<#Brittáin>
+    a foaf:Person;
+    irishRel:genName "Brittáin";
+    irishRel:nomName "Brittán";
+    rdfs:comment "Fine Cilli Bicsige";
+    rel:descendantOf <#Conall>.
+
+<#Meldae>
+    a foaf:Person;
+    irishRel:genName "Meldae";
+    rdfs:comment "Fine Cilli Bicsige";
+    rel:descendantOf <#Conall>.
+
+<#Chúáin>
+    a foaf:Person;
+    irishRel:genName "Chúáin";
+    irishRel:nomName "Cúán";
+    rel:descendantOf <#Conall>;
+    rdfs:comment "Uí Chúáin [...] h-i Fobur".
+
+<#Draignén>
+    a foaf:Person;
+    irishRel:genName "Draignén";
+    irishRel:nomName "Draigniu";
+    rdfs:comment "Úi Draignén h-i Fobur";
+    rel:descendantOf <#Conall>.
+
+<#Sellaig>
+    a foaf:Person;
+    irishRel:genName "Sellaig";
+    irishRel:nomName "Sellach";
+    rdfs:comment "Úi Sellaig [...] i m-Maig Lethan";
+
+<#Forca>
+    a foaf:Person;
+    irishRel:genName "Forca";
+    rdfs:comment "Úi Forca i m-Maig Lethan";
+    rel:descendantOf <#Conall>.
+
+<#Inmete>
+    a foaf:Person;
+    irishRel:genName "Inmete";
+    rdfs:comment "Úi Inmete h-i Connull";
+    rel:descendantOf <#Conall>.
+# owl:sameAs <#EchachInmete>? EPT
+
+<#NathÍ>
+    a foaf:Person;
+    irishRel:genName "Nath Í";
+    irishRel:nomName "Nath Í";
+    rdfs:comment "Cenél Nath Í i m-Maigib Ailbe ó Chill Éogain, is leo grian na cille";
+    rel:descendantOf <#Conall>.
+
+<#FindFile-a68034e0>
+    a foaf:Person;
+    irishRel:nomName "Find File";
+    owl:sameAs <#FindFili>;
+    rel:childOf <#RossaRuaid-dd99e390>;
+    rdfs:comment "is é senathair Lagen olchena".
+
+<#RossaRuaid-dd99e390>
+    a foaf:Person;
+    irishRel:genName "Rossa Ruaid";
+    irishRel:nomName "Ros Ruad";
+    owl:sameAs <#Rossa>.
+
+<#ConchobarArbatruad-13961450>
+    a foaf:Person;
+    irishRel:nomName "Conchobar Arbatruad";
+    rel:childOf <#FindFile-a68034e0>;
+    owl:sameAs <http://example.com/LL/lagin.ttl#ConchobarArbatruad>.
+
+<#ConChorb>
+    a foaf:Person;
+    irishRel:genName "Con Corb";
+    irishRel:nomName "Cú Chorb";
+    rel:descendantOf <#ConchobarArbatruad-13961450>;
+    owl:sameAs <http://example.com/LL/lagin.ttl#CúChorbb>;
+    irishRel:numChild 4.
+
+<#NioCorb>
+    a foaf:Person;
+    irishRel:nomName "Nio Corb";
+    rel:childOf <#ConChorb>;
+    owl:sameAs <http://example.com/LL/lagin.ttl#NiaCorbb>;
+    rdfs:comment "Dál Niad Cuirp dia tát mc Cathaír Máir".
+
+<#CathaírMáir>
+    a foaf:Person;
```

Comment:
Yes, it is. I thought I linked back to a previous appearance of his in Rawl.B.502, which in turn links to LL. But, just in case, I have added an LL link here too.